Assets/HoloToolkit/SpectatorView/Scripts/Networking/SpectatorViewNetworkDiscovery.cs

```diff
@@ -0,0 +1,275 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Linq;
```

Comment:
What are we using linq for?

Reply:
if it is not being used, it should be removed.